css/at-rules/font-face.json

```diff
@@ -0,0 +1,283 @@
+                "version_added": true
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": "3.5"
+              },
+              "firefox_android": {
+                "version_added": "4"
+              },
+              "ie": {
+                "version_added": "9"
+              },
+              "ie_mobile": {
+                "version_added": "10"
+              },
+              "opera": {
+                "version_added": "11.1"
+              },
+              "opera_android": {
+                "version_added": "11"
+              },
+              "safari": {
+                "version_added": "5.1"
+              },
+              "safari_ios": {
+                "version_added": "5"
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "WOFF_2": {
+          "__compat": {
+            "description": "WOFF 2",
+            "support": {
+              "webview_android": {
+                "version_added": "36"
+              },
+              "chrome": {
+                "version_added": "36"
+              },
+              "chrome_android": {
+                "version_added": "36"
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": [
+                {
+                  "version_added": "39"
+                },
+                {
+                  "version_added": "35",
+                  "version_removed": "38",
+                  "flag": {
+                    "type": "preference",
+                    "name": "gfx.downloadable_fonts.woff2.enabled",
+                    "value_to_set": "true"
+                  }
+                }
+              ],
+              "firefox_android": [
+                {
+                  "version_added": "39"
+                },
+                {
+                  "version_added": "35",
+                  "version_removed": "38",
+                  "flag": {
+                    "type": "preference",
+                    "name": "gfx.downloadable_fonts.woff2.enabled",
+                    "value_to_set": "true"
+                  }
+                }
+              ],
+              "ie": {
+                "version_added": false
+              },
+              "ie_mobile": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": "23"
+              },
+              "opera_android": {
+                "version_added": "23"
+              },
+              "safari": {
+                "version_added": false
```

Comment:
"10" with a note "Supported only in Mac OS 10.12 (Sierra) and later"

See https://developer.apple.com/library/content/releasenotes/General/WhatsNewInSafari/Articles/Safari_10_0.html#//apple_ref/doc/uid/TP40014305-CH11-SW1